### PR TITLE
Add tooltip for non-compliant saptune

### PR DESCRIPTION
**Pull request by rtorrero**
# Description

This PR adds a tooltip to add further details when saptune is not compliant

Preview:
Captura desde 2024-02-15 15-37-12

### CODE REVIEW

File: assets/js/pages/SaptuneDetails/SaptuneTuningState.jsx [hunk 1]
```diff
@@ -9,7 +10,12 @@ function SaptuneTuningState({ state }) {
     case 'not compliant':
       return (
         <div className="flex">
-          <HealthIcon health="critical" />
+          <Tooltip
+            content="Run `saptune note verify` in the host for further details"
```
Comment (arbulu89) [inline assets/js/pages/SaptuneDetails/SaptuneTuningState.jsx:14]: I don't think markdown works here as it is. You would need to wrap it up, something like:
```
<ReactMarkdown className="markdown" remarkPlugins={[remarkGfm]}>
Run `saptune note verify` in the host for further details
 </ReactMarkdown>
```

**Review by arbulu89 — COMMENTED** [review]
Just a comment.
If @jagabomb agrees that the tooltip needs to go in the bottom is ok from my side

**Comment by jagabomb** [comment]
> Just a comment. If @jagabomb agrees that the tooltip needs to go in the bottom is ok from my side

Something like this would be good...
Screenshot 2024-02-15 at 17 16 35

**Review by arbulu89 — APPROVED** [review]
If @jagabomb is happy, I'm happy.
Jurgen, you could check it by yourself in saptune details view storybook, but you would need to change the tuning value to `not compliant` in the options

**Comment by rtorrero** [comment]
As discussed with @arbulu89 we ended up removing the markdown as it adds very little value and looks quite odd

**Review by EMaksy — APPROVED** [review]
LGTM!

**Review by CDimonaco — APPROVED** [review]

**Comment by jagabomb** [comment]
@rtorrero is it possible to wrap the text over two lines like screenshot below instead of one long line?
Screenshot 2024-02-15 at 18 14 02

**Comment by rtorrero** [comment]
@jagabomb should be good now: 
Captura desde 2024-02-16 08-54-12

**Review by jagabomb — APPROVED** [review]
LGTM!

**Pull request merged**